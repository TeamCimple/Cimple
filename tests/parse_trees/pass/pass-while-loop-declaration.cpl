int main() {
	int i = 1;

	while(1) {
		printf("OH BOY\n");
	}
	return 0;
}
