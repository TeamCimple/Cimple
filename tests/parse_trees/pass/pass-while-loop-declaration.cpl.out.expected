FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations: (int EQUALS(Var(i) Int(1),),  StatementList: WHILE Int(1) CompoundStatement(Declarations:  StatementList: Statement(Call(Receiver()FunctionName: Identifier(printf) Params: String("OH BOY\n")))), RETURN(Int(0)))) 
